data/remnant/remnant missions.txt

```diff
@@ -4024,6 +4051,33 @@ mission "Remnant: Cognizance 14"
 			`As you arrive back at Viminal, you broadcast a warning to the port authorities that something is following you. You inform them that it is not hostile, but it is an unknown entity that originated in the new system.`
 
 
+mission "Remnant: Bounty 4"
+	job
+	repeat
+	name "Remnant bounty"
+	description "Hunt down a Korath ship that is lurking in Remnant territory, then return to <planet> to receive your payment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Cognizance 14: done"
+		random < 60
```

Comment:
Dredger bounties ought to be rarer than Raider bounties.

```suggestion
		random < 20
```

Reply:
This is a good point. However, I think 20 is too low. Should we try for 30 or 40?